File: 프로그래머스/lv0/120894. 영어가 싫어요/영어가 싫어요.java
class Solution {
    public long solution(String numbers) {
        long answer = 0;
        String[] alp = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        for(int i=0; i<=9; i++){
            numbers = numbers.replaceAll(alp[i],i+"");
        }
        answer = Long.parseLong(numbers);
        return answer;
    }
}

File: 프로그래머스/lv0/120894. 영어가 싫어요/영어가 싫어요.js
function solution(numbers) {
    var answer = numbers;
    var num = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
    for(var i=0; i<10; i++){
        answer = answer.replaceAll(num[i], i);
    }
    return parseInt(answer);
}